h5p-bildetema/semantics.json

```diff
@@ -270,5 +270,117 @@
         "type": "text"
       }
     ]
+  },
+  {
+    "label": "Default languages",
+    "description": "There should be up to three default languages. These are defaults for the current site language. We recommend adding three of them",
+    "name": "defaultLanguages",
+    "type": "list",
+    "entity": "Language",
+    "field": {
+      "description": "Language code should be on <a href=https://en.wikipedia.org/wiki/List_of_ISO_639-3_codes>ISO-639-3</a> format",
```

Comment:
Maybe change label name and description now that the "language code" is not used